docs/UserGuide.md

```diff
@@ -547,100 +537,88 @@ Figure 5.3.3.1. Example of edit expenditure command
 
 ![Example of find expenditures command](images/CommandScreenShots/5_3_4_1_findExpenditures.png)
 
-Figure 5.3.5.1. Example of find expenditures command
+Figure 5.3.4.1. Example of using the find expenditures command.
 
 #### 5.3.5. Listing expenditures: `list`
 (Contributed by Chin Hui)
 
-You can use this command to **list all expenditures** in the current budget.
-
-This command is usually used to display all expenditures after searching for specific expenditures using the `find` command.
+Format: `list`
 
-> 📕️ If the `find` command has not been used, the `list` command has no visible effect.
+You can use this command to list all expenditures in the current budget.
 
-Format: list
+When you list expenditures, the list view will display all expenditures belonging to the current budget.
 
-✏ Example:
+ > 📕 This command is usually used after using the [find expenditure command](#534-finding-expenditures-find).
 
-`list`
+✏ Example: `list`
 
-This will display all existing expenditures in the current budget.
+This will display all existing expenditures in the current budget as seen in Figure 5.3.5.1 below:
 
 ![Example of list expenditures command](images/CommandScreenShots/5_3_5_1_listExpenditures.png)
 
-Figure 5.3.5.1. Example of list expenditures command
+Figure 5.3.5.1. Example of using the list expenditures command.
 
 #### 5.3.6. Sorting expenditures: `sort`
 (Contributed by Yu Ming)
 
-You can use this command to **sort your expenditures** within a budget.
-
-You can sort your expenditures within a budget based on two different type of sorts:
-1. Sort by added date (Ordered from most recently created to least recently created)
+Format: `sort TYPE`
 
-    > 📕 Note: If expenditures are created on the same date. They will be sorted by name within themselves.
-   
-2. Sort by name of the expenditure (Ordered in alphabetical order (case-insensitive))
+ Parameters | Requirement | Comments
+ ---------- | ----------- | ------
+ TYPE       | Required    | Types of sort: `time`, `name`
 
-    > ⚠ If there are no expenditures or only one expenditure in the budget, the sort command will not have 
-    any effect. 
+You can use this command to sort your expenditures (within a budget) by the given `TYPE`.
 
-Format: `sort TYPE`
+There are a two ways you can sort your expenditures:
 
- Parameters | Requirement | Comments
- ------------ | ------------- | ------
- TYPE | Required | Types of sort: `time`, `name`
+- By their creation date with the most recently created budget at the top.
 
-✏ Example:
+> 📕 Budgets that are created on the same date will be sorted in alphabetical order.
+    
+- By their name in alphabetical order.
 
-`sort time`
+✏ Example: `sort name`
 
-This will sort all the expenditures in that budget by their added date with the latest expenditure added at the top.
+This will sort the expenditures in that budget (that are currently displayed) by their name in alphabetical order as seen in Figure 5.3.6.1 below:
```

Comment:
"This will sort the expenditures (that are currently displayed) by their name in alphabetical order as seen in Figure 5.3.6.1 below:"